intermediate-python-debugging/problem-set-using-the-debugger.md

```diff
@@ -85,50 +85,49 @@ Repeat this process until the program stops crashing, and the bug is fixed. Duri
 ### Second Bug: Debugging a Failing Test
 
 One of the tests is failing, find the failing test in `tests/test_order.py`.  
+![Failing test](../assets/vs-code-debugger/exercise-failing-test.png)  
+*Fig. An example of a failing test in VS Code. Since the test is failing, the status icon is displayed as a red circled X.*
 
-![Failing test](../assets/vs-code-debugger/exercise-failing-test.png)
-*Fig. An example of a failing test in VS Code*
-
-Create a breakpoint in the failing test `test_calculate_total_with_multiple_products` and use the step-into and step over commands to trace through the code and see why the test is failing.  You can click on the `Debug Test` link above the test function to run that one test in the debugger.
+Create a breakpoint in the failing test `test_calculate_total_with_multiple_products` and use the step-into and step over commands to trace through the code and see why the test is failing.  You can right-click the status icon positioned to the left of the test name to reveal the `Debug Test` option. Click it to run that one test in the debugger.
```

Comment:
Do we need 2 spaces between each sentence in a paragraph for Learn's formatting? Asking since that seems to be the practice across the file, but there's only 1 space in front of the last sentence.

Reply:
Learn only cares about 2 spaces at the end of a line for wrapping hinting. But I can add in a few spaces where I missed them for consistancy. I've fought so hard to overcome my 2 spaces after a full stop from typing class, but somehow it became the style in our MD files. 😩